projects/control-service/cicd/.gitlab-ci.yml

```diff
@@ -249,7 +250,7 @@ control_service_vdk_heartbeat_release:
   stage: release
   script:
     - export CHART_NAME=pipelines-control-service
-    - export CHART_VERSION=$(cat projects/control-service/projects/helm_charts/$CHART_NAME/version.txt)
+    - export CHART_VERSION=$(cat projects/control-service/projects/helm_charts/$CHART_NAME/version.txt).$CI_PIPELINE_ID
```

Comment:
How are we sure someone will not set patch version in the version.txt file e.g 1.2.0  next time they edit it. 

I'd parse out the major and minor version (as it is in other projects) to be on the safe side.